Makefile

```diff
@@ -4,8 +4,9 @@ ROOTDIR=$(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 # Base path used to install.
 DESTDIR=/usr/local
 
-# Used to populate version variable in main package.
-VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
```

Comment:
Don't replace this: this will always take a tag or form a tag, with the git commit, relative to a tag.